bindings/pyroot_experimental/PyROOT/python/ROOT/pythonization/_tcollection.py

```diff
@@ -42,12 +41,9 @@ def _count_pyz(self, o):
     # - Number of occurrences of the object in the collection
     n = 0
 
-    it = TIter(self)
-    obj = it.Next()
-    while obj:
-        if obj == o:
+    for elem in self:
```

Comment:
This works. Nevertheless, Python loops are known to be slow. Can we maybe express those in a way in which the C layer is directly kicking in?
Perhaps this is not crucial now and here, but since we are re-writing everything...

Reply:
I tried with a list comprehension syntax and the times are comparable (it must be doing a bytecode loop anyway). Also the `reduce` method takes more or less the same (probably because of the function calls). Any other ideas?